Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM arm64v8/golang:alpine3.12 AS builder
+FROM arm64v8/golang:alpine AS builder
```

Comment:
:question: No longer locking the version of alpine, why

Reply:
Latest is working, we can lock once a new alpine breaks the build

Reply:
Can lock to 3.16.0 if you think that's better